### PR TITLE
fix: merge node config allows overide

### PR DESCRIPTION
**Pull request by alaibe**

### CODE REVIEW

File: api/geth_backend.go [hunk 1]
```diff
@@ -350,8 +350,8 @@ func (b *GethStatusBackend) mergeConfig(n *params.NodeConfig) (*params.NodeConfi
 		return nil, err
 	}
 
-	// Overwrite db configuration (only adds new values)
-	if err := mergo.Merge(conf, n); err != nil {
+	// Overwrite db configuration
+	if err := mergo.MergeWithOverwrite(conf, n); err != nil {
```
Comment (richard-ramos) [inline api/geth_backend.go]: MergeWithOverwrite is deprecated
```suggestion
	if err := mergo.Merge(conf, n, mergo.WithOverride); err != nil {
```

**Review by richard-ramos — APPROVED** [review]

**Comment by churik** [comment]
Is it ready for client testing @alaibe ? Any possible regression on clients side (desktop, mobile)?

**Comment by alaibe** [comment]
@churik yes it is ready, on mobile I don't know, on desktop no

**Comment by alaibe** [comment]
@churik has it been tested?

**Comment by churik** [comment]
not yet. will be tested shortly

**Pull request merged**